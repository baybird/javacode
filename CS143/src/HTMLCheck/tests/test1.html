<b><i><br/></b></i>
