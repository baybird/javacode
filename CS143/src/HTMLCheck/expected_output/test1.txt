<b><i><br /></i></b>